content/english/javascript-basics/Arrays/Methods.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Methods"
+date: 2020-07-28T11:45:38-07:00
+draft: false
+weight: 8
+---
+
+To make things easier, arrays provide a lot of methods. Let's go over some of the basic ones which would give you an idea of they look like.
```

Comment:
Explain further what is a method. For example - "a method represent some action we wish to perform on the array."

Also explain how is this different from a property.